### DISCUSSION TITLE
SEB with specific software (coach) not running

### DISCUSSION THREAD

**Discussion by rmjonkeredu** [SafeExamBrowser/seb-win-refactoring#485] [Q&A] [answered]
We are trying to set up a physics exam for modelling and video measuring using the software Coach ([www.cma-science.nl](https://cma-science.nl/coach7_desktop_downloads_en).

We try to white list this software as we do with word or excel or other 'simple' software. This program however provides problems.

When adding this to the permitted software the student has to log in a second time in the testing environment (woots.nl). Somehow the student gets kicked from the system. This is not a major issue as the student can log in again.

For the software I added the full path in the field for executable (which also works for software like Claroread). When starting the test, SEB is started, students logs in again and can start the test. The icon of the software coach is shown in the task bar, it shortly shows and disappears to never come back. All other permitted software works, and the exam works. The required software does not.

When contacting the helpdesk of the software, they suggested that a second executable is started to change the text based on the language settings. We added this program to the permitted software and now the exam doesn't start at all due to 'false configuration'.

We would be very happy with a clue on how to fix this.

**Comment by dbuechel** [comment, marked as answer]
You'll probably need to allow the second executable as well in the SEB configuration, but still start the software using the first (main) executable (see also https://safeexambrowser.org/windows/win_usermanual_en.html#ApplicationsPane). SEB 3.x provides a real-time application log (see Security > Allow access to application log) where you can test your configuration and see any errors and the status of SEB's application monitoring in real time.
